hostapd/ctrl_iface.c

```diff
@@ -3567,6 +3578,7 @@ static int hostapd_ctrl_iface_receive_process(struct hostapd_data *hapd,
 	} else if (os_strncmp(buf, "STA-NEXT ", 9) == 0) {
 		reply_len = hostapd_ctrl_iface_sta_next(hapd, buf + 9, reply,
 							reply_size);
+#ifndef __ZEPHYR__
```

Comment:
These are needed for monitoring internal events and are already supported for WPA supplicant, but if you want can add a TODO so that at least we are aware of the pending item?

Reply:
Added TODO to track this pending item.